### DISCUSSION TITLE
Support for Cineform RAW video recording

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#945] [Feature Requests]
Would be an amazing feature to get compressed Cineform RAW video encoding on GoPro Labs firmware.

**Comment by dnewman-gpsw** [comment]
You already have it. This the format of .GPR files.  But as the CPU is only encoding raw a ~1fps, it can't be applied to video.

**Reply by RikshaDriver** [reply]
Aware of the GPR VC-5/Cineform compression for RAW photos but this is for Videos in particular. There is a general trend moving towards compressed RAW video and the CineForm RAW format is arguably superior but hasn't had much love in practical RAW video implementations.

Is the hurdle on GoPro HEROs purely down to CPU limitations for compression and x86/x64 specific optimizations?


BTW on a side note, there is/was an issue with gpr_tools and HERO12 where it couldn't convert HERO12 GPR source files to other formats. This was with a 1.x firmware release and haven't had a chance to test for a while with the newer firmware, but it may still be an issue.

**Reply by dnewman-gpsw** [reply]
Yes, it is only CPU limitation. VC5 is fully ARM optimized on camera, is it simply a little camera. Camera that are doing ProRES Raw or BMD Raw are bigger, more power dissipation it supported.

**Reply by RikshaDriver** [reply]
FYI regarding this issue:

> BTW on a side note, there is/was an issue with gpr_tools and HERO12 where it couldn't convert HERO12 GPR source files to other formats. This was with a 1.x firmware release and haven't had a chance to test for a while with the newer firmware, but it may still be an issue.

I have raised the following issue: https://github.com/gopro/gpr/issues/49
